libs/s25main/desktops/dskSelectMap.cpp

```diff
@@ -172,12 +176,34 @@ void dskSelectMap::Msg_OptionGroupChange(const unsigned /*ctrl_id*/, unsigned se
         FillTable(ListDir(worldsPath, "wld"));
     }
 
+    if(brokenMapPaths.size() > numFaultyMapsPrior)
+    {
+        std::string errorTxt =
+          helpers::format(_("%1% map(s) could not be loaded. Check the log for details"), brokenMapPaths.size() - numFaultyMapsPrior);
```

Comment:
should be logged to the log too

Reply:
The log already contains a message for each map. IMO logging this would be redundant